### PR TITLE
[Bug] LPS-170859

### PR DESCRIPTION
**Pull request by yuhai**
@tinatian, @dantewang, the previous mentioned issue(layoututilitypageentry data won't be deleted from https://github.com/liferay-core-infra/liferay-portal/pull/3727) doesn't happen. Because LPS-176776 added one flag to avoid creating the data when creating a company. I asked Fiona(who is from echo team in Dalian) and she told me the developer will remove the flag and use the final solution to fix the issue.  We may ignore the issue. 

The pr fixed the issue  "The leftover data is com.liferay.view.count.model.ViewCountEntry".

### CODE REVIEW

**Comment by Kvale1733** [comment]
@yuhai No regressions. com.liferay.company.service.test.CompanyLocalServiceTest Passes on this PR.

**Comment by tinatian** [comment]
@yuhai For https://github.com/liferay-core-infra/liferay-portal/pull/3759/commits/fa738c27b27d9c4b9cbc1730a72fc199c86bea10, it sounds like an issue you can see when BackgroundTask running on anther thread. If yes, we should fix the root issue rather than avoiding it in the test

**Comment by tinatian** [comment]
@yuhai Please verify if https://github.com/yuhai/liferay-portal/pull/315 fixed the ViewCountEntry issue.

File: modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java [hunk 1]
```diff
@@ -287,12 +289,14 @@ public void testAddAndDeleteCompanyWithDLFileEntryTypes() throws Exception {
...
 
-		_dlAppLocalService.addFileEntry(
+		FileEntry fileEntry = _dlAppLocalService.addFileEntry(
 			null, userId, guestGroup.getGroupId(), 0, "test.xml", "text/xml",
 			"test.xml", "", "", "", "test".getBytes(), null, null,
 			serviceContext);
 
 		_companyLocalService.deleteCompany(companyId);
+
+		_dlAppLocalService.getFileEntry(fileEntry.getFileEntryId());
```
Comment (tinatian) [inline modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java:301]: @yuhai What are you asserting here?

**Comment by tinatian** [comment]
@yuhai For this commit https://github.com/liferay-core-infra/liferay-portal/pull/3759/commits/bc71b3cc20add6cfa33c6578f0b9b244daf96c72, please make the company is a local variable with @DeleteAfterTestRun, this way it will remove the company even when an exception thrown.

**Comment by tinatian** [comment]
@yuhai Please see my comments and resend.

Thanks.
Tina

**Pull request closed without merging**